sources/whippet/src/policy.rs

```diff
@@ -0,0 +1,861 @@
+use crate::error::{self, Result};
+use indexmap::IndexMap;
+use serde::Deserialize;
+use snafu::{ensure, ResultExt};
+use std::fs;
+use std::path::Path;
+
+/// Default directory path additional policies are read from
+pub(crate) const DBUS_POLICY_DROPINS_PATH: &str = "/usr/share/dbus-1/system.d";
+/// Multiplier used to calculate the priority of a rule
+/// https://github.com/bus1/dbus-broker/blob/b0db0890d1254477cf832e5f9f0a798360c80fd9/src/launch/policy.c#L237
+const DEFAULT_POLICY_CONTEXT_MULTIPLIER: u64 = 1;
+const DEFAULT_POLICY_CONTEXT_USER: u64 = 3;
+const PRIORITY_BASE: u64 = u64::MAX / 7;
+
+/// Represents a minimal policy configuration for the dbus-broker. Each section on the policy is
+/// known as a Context.
+#[derive(Debug, Deserialize, Default)]
+#[serde(deny_unknown_fields)]
+pub(crate) struct Policy {
+    /// The default rules that apply to all contexts. This maps to the original <policy
+    /// context="default">
+    pub(crate) default: Option<Context>,
+    /// User-specific rules. This maps to the original <policy user="<user>">
+    pub(crate) user: Option<IndexMap<String, Context>>,
+}
```

Comment:
These comments aren't very meaningful. Why is each section on the policy known as a `Context`? Does that matter? Why do I need to know what the struct fields map to in terms of XML fragments?